useraudit/models.py

```diff
@@ -1,4 +1,5 @@
 from django.db import models
+from django.utils import datetime
```

Comment:
I like the change but shouldn't this be?

    from django.utils import timezone